easybuild/easyconfigs/s/spglib-python/spglib-python-2.1.0-foss-2023a.eb

```
@@ -0,0 +1,29 @@
+easyblock = 'PythonBundle'
+
+name = 'spglib-python'
+version = '2.1.0'
+
+homepage = 'https://pypi.python.org/pypi/spglib'
+description = "Spglib for Python. Spglib is a library for finding and handling crystal symmetries written in C."
+
+toolchain = {'name': 'foss', 'version': '2023a'}
```

Since `spglib-python` has no MPI dependecy we can place this in the `gfbf` toolchain:
```suggestion
toolchain = {'name': 'gfbf', 'version': '2023a'}
```